### ISSUE TITLE
Support custom time periods

### ISSUE THREAD

**Issue by dbaelz** [dbaelz/OnOff-Tracker#1]
- Define freely selectable time periods (from-until) including a title
- Create, add and delete them as cards on the list

**Comment by brixtonasias** [comment]
Good idea. Would like to have like three default ones: Today, Week, Total.

**Comment by dbaelz** [comment]
Add new card: Last seven days. Custom time periods moved to Milestone 1.2.
